modules/ROOT/pages/monitoring.adoc

```diff
@@ -1,57 +1,15 @@
-= Monitoring
+= Monitoring Applications and Servers
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: monitoring, runtime manager, arm, cloudhub, applications, servers
-
-image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
-image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-active.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
-
-After you deploy your application, following any of the xref:deployment-strategies.adoc[deployment strategies] supported by the Runtime Manager, you can use the Runtime Manager to monitor its performance and that of the server it runs on (if you deployed on-premises).
-
-[NOTE]
-Depending on the xref:deployment-strategies.adoc[deployment strategy] you use, there is a different set of possible monitoring resources available. This feature gap will be gradually reduced as new releases implement the missing features on either side.
-
-
-== Monitoring Applications
-
-You can access monitoring through the following tools:
-
-* xref:alerts-on-runtime-manager.adoc[Alerts] – image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
-image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"] Send e-mail alerts whenever key events occur. A history of when these alerts were triggered can also be accessed.
 
+After you deploy your application, you can use Runtime Manager to monitor the application's performance. If you deployed the application to an on-premises server, you can also monitor the server. 
```

Comment:
delete "you can"

Reply:
According to Salesforce style guide, using "can" is acceptable. It means the ability to do something.